luatest/replica_set.lua

```diff
@@ -183,12 +183,16 @@ function ReplicaSet:stop()
     end
 end
 
---- Stop all servers in the replica set and clean their working directories.
+--- Stop all servers in the replica set and save their artifacts if the test fails.
+-- This function should be used only at the end of the test (after_test,
```

Comment:
Please wrap `after_test` into backticks. The same for `after_each` and `after_all`.

Reply:
Fixed